kolibri/plugins/html5_app_renderer/assets/src/views/index.vue

```diff
@@ -83,6 +83,8 @@
     height: 100vh
     max-height: calc(100vh - 24em)
     min-height: 400px
+    overflow-x: auto
```

Comment:
Confirmed, no difference in behaviour on browser stack between `scroll` and `auto` - would prefer `auto` as it means that scroll bars are only displayed when they are needed.